app/lib/actions/katello/content_view/destroy.rb

```diff
@@ -12,8 +12,10 @@ def plan(content_view, options = {})
 
           sequence do
             concurrence do
-              content_view.content_view_versions.each do |version|
-                plan_action(ContentViewVersion::Destroy, version, options)
+              unless options.fetch(:force_destory_cv, false)
```

Comment:
instead of doing an unless, what if the versions to destroy were passed in as an option (that could default to 'content_view.content_view_versions'.  I would put something like this at the top:
```
cvvs_to_destroy = options.fetch(:deletable_content_view_versions, content_view.content_view_versions)
```

Reply:
then the force flag isn't needed

Reply:
and the Remove action could call this and pass in :deletable_content_view_versions => []

Reply:
Updated to move pieces to remove action entirely with a detroy_cv param.